routes/api/v1/cruises.js

```diff
@@ -210,29 +210,31 @@ exports.plugin = {
 
         const query = {};
 
-        //Hidden filtering
+        // Hidden filtering
         if (typeof request.query.hidden !== "undefined") {
-
-          if (request.auth.credentials.scope.includes('admin')) {
+          // If authenticated and admin, allow filtering by hidden
+          if (request.auth.credentials && request.auth.credentials.scope.includes('admin')) {
             query.cruise_hidden = request.query.hidden;
           }
+          // If trying to access hidden but not authenticated or not admin
           else if (request.query.hidden) {
             return Boom.unauthorized('User not authorized to retrieve hidden cruises');
           }
+          // Explicitly requesting non-hidden
           else {
             query.cruise_hidden = false;
           }
         }
         else {
-          if (!request.auth.credentials.scope.includes('admin')) {
+          // No hidden param specified - default to non-hidden for non-admins or unauthenticated
+          if (!request.auth.credentials || !request.auth.credentials.scope.includes('admin')) {
             query.cruise_hidden = false;
           }
         }
```

Comment:
I think this logic (which predates this change) is a little convoluted... Is this equivalent?

```js
let canQueryHidden = request.auth.credentials?.scope.includes('admin')) || false;
let wantsQueryHidden = request.query.hidden === true;
query.cruise_hidden = canQueryHidden && wantsQueryHidden;
if (wantsQueryHidden && !canQueryHidden) {
    return Boom.unauthorized('User not authorized to retrieve hidden cruises');
}
```